modules/options-for-notebook-server-environments.adoc

```diff
@@ -19,7 +19,9 @@ endif::[]
 ifndef::upstream[]
 [NOTE]
 --
-Notebook images are supported for a minimum of one year. Major updates to pre-configured notebook images occur about every six months. Therefore, two supported notebook image versions are typically available at any given time. Legacy notebook image versions, that is, not the two most recent versions, might still be available for selection. Legacy image versions include a label that indicates the image is out-of-date. To use the latest package versions, {org-name} recommends that you use the most recently added notebook image.
+Notebook images are supported for a minimum of one year. Major updates to pre-configured notebook images occur about every six months. Therefore, two supported notebook image versions are typically available at any given time. Legacy notebook image versions, that is, not the two most recent versions, might still be available for selection. Legacy image versions include a label that indicates the image is out-of-date. 
+
+Version 1.2 of notebook images available in {productname-short} 2.4 will not longer be supported at {productname-short} 2.5. To use the latest package versions, {org-name} recommends that you use the most recently added notebook image, that is, notebook images with the version 2023.2. 
```

Comment:
Suggestion:

_Version 1.2 of notebook images available in {productname-short} 2.4 will no longer be supported in {productname-short} 2.5. To use the latest package versions, {org-name} recommends that you use the most recently added notebook image._


- 'not longer' to 'no longer'
- 'at' to 'in', to match wording earlier in the line. (Alternatively, could use 'from', if we want to communicate that this will be the case from 2.5 onwards.)

- I'd leave out 'that is, notebook images with the version 2023.2' - I think flagging that 1.2 won't be available to select is enough alongside just maintaining the 'latest' recommendation.